src/lib/Color.ts

```diff
@@ -0,0 +1,45 @@
+const colors =[
```

Comment:
It seems like these aren't exactly the original colors, and there are some dupes

Reply:
I have removed the duplicate color. hadn't check it sorry.